public/app/features/admin/state/actions.ts

```diff
@@ -134,6 +135,10 @@ export function deleteOrgUser(userId: number, orgId: number): ThunkResult<void>
 
 export function loadUserSessions(userId: number): ThunkResult<void> {
   return async (dispatch) => {
+    if (!contextSrv.hasPermission(AccessControlAction.UsersAuthTokenList)) {
```

Comment:
Not 100% sure we need these extra checks in the actions. The UI elements are already disabled for them and if users somehow still manage to execute the action, the API will return an error. With these checks if users manage to execute an action (e.g. press a button), that they don't have authorisation for, they won't get a proper feedback. I'm thinking mostly about cases when due to some bug UI elements aren't disabled properly, as an example.

Reply:
Yes, good point. That makes me feel weird a bit since action can be executed, but it's skipped with no result. User can think that's a bug.

Reply:
We only need extra checks for actions that required for loading state for UI. For instance, this `loadUserSessions` call invoked even sessions table is not rendering, so we want to prevent this if user has no access to call this endpoint (has no UsersAuthTokenList permission).